docs/blog/posts/2023-04-20-introducing-trusted-publishers.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Introducing 'Trusted Publishers'
+description: Announcing a new, more secure way to publish to PyPI
+author: Dustin Ingram
+publish_date: 2024-04-20
+tags:
+  - publishing
+  - security
+  - oidc
+---
+
+Starting today, PyPI maintainers can adopt a new, more secure publishing method
```

Comment:
```suggestion
Starting today, PyPI package maintainers can adopt a new, more secure publishing method
```

To denote a difference between folks who maintain PyPI and those who maintain packages published to PyPI.

Alternately, "PyPI publishers"?